src/main/java/com/volasoftware/tinder/services/implementations/AuthenticationServiceImpl.java

```diff
@@ -155,6 +156,19 @@ public AccountDto recoverPassword(Principal principal) {
         return accountMapper.accountToAccountDto(savedAccount);
     }
 
+    @Override
+    public EmailDto reverify(EmailDto emailDto) {
```

Comment:
Why do methods like this and regenerateToken exist? Are there methods that do the same thing? If there are, it's appropriate to reuse them. If they do not exist, then you need to change the method's names

Reply:
The methods are not doing the same thing, so the were renamed.